hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -900,6 +901,173 @@
+            - Write the pin allowed ctl register to allow some of the pins to be overridden with
+              either 0 or 1 or both.
+            - Write the pin out ctrl register to enable the override on some of the pins.
+            - Write the pin out values register to set a pattern on those pins.
+            - Verify that at the chip IO outputs, pins on which override should be active is
+              reflecting the overridden values. All others should reflect the values driven on chip
+              inputs.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sysrst_ctrl_in_irq
+      desc: '''Verify the SYSRST ctrl can detect an input combination to signal in interrupt.
+
+            - Program a specific combination of transitions on pwrb, key*, ac_present and ec_reset_l
+              pins to trigger an interrupt by writing to key_intr_ctl register.
+            - Program the key_intr_debounce_ctl register to debounce an appropriate time.
+            - Enable the interrupt at sysrst ctrl as well as at the PLIC.
+            - Glitch the inputs at the chip IOs before stabilizing on the programmed transitions.
+            - SW services the interrupt when triggered, verifies the pin input value and
+              key_intr_status for correctness and clears the interrupt status.
+            - Verify separately, eack key combination sufccessfully generates an interrupt.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sysrst_ctrl_sleep_gsc_wakeup
+      desc: '''Verify the SYSRST ctrl can wake up the chip from deep sleep.
+
+            - Read the reset cause register in rstmgr to confirm we are in POR reset phase.
+            - Program one of the com_sel_ctl_* CSRs to choose a set of inputs to be detected as
+              a low power wake up signal.
+            - Program the associated detection timer.
+            - Program the detection outcome CSR's (com_out_ctl) interrupt bit to 1.
+            - Program the pwrmgr to put the chip in deep sleep state and wake up on GSC wake up
+              event.
+            - Issue a WFI to bring the chip in low power state.
+            - After the chip has entered low power mode, set the sysrst ctrl inputs at the chip IOs
+              to the programmed combination for the duration of the detection timer.
+            - Read the reset cause register to confirm wake up from low power exit phase.
+            - Read the pwrmgr wake up status register to confirm GSC wake up.
+            - Read the pin input value and the combo_intr_status CSRs to verify the correct
+              combination on inputs woke up the chip from sleep.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sysrst_ctrl_gsc_reset
+      desc: '''Verify the SYSRST ctrl can reset the chip from normal state.
+
+            - Read the reset cause register in rstmgr to confirm we are in POR reset phase.
```

Comment:
Same as above... Imagine a version of this test run within a collection of other tests.